adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -513,7 +514,15 @@ private boolean checkIfBrokerHasLltChanges() {
         } catch (PackageManager.NameNotFoundException ignored) {
         }
 
-        return authVersionCode >= AUTHENTICATOR_LLT_VERSION_CODE && cpVersionCode >= CP_LLT_VERSION_CODE;
+        try {
+            PackageInfo brokerHostPackageInfo = packageManager.getPackageInfo(AuthenticationConstants.Broker.BROKER_HOST_APP_PACKAGE_NAME, 0);
+            brokerHostVersionCode = brokerHostPackageInfo.versionCode;
+        } catch (PackageManager.NameNotFoundException ignored) {
```

Comment:
Don't we at least want to log at the info level?

Reply:
This change lifted verbatim from @shahzaibj's changes in e42ed621a403f40b3659dd2c57cf0dcd4e26a229 to unjam the build - not looking to apply further changes as all I'm doing is pulling into this branch for deployment